data/presets/presets/waterway/dock.json

```diff
@@ -0,0 +1,11 @@
+{
+    "geometry": [
```

Comment:
also add to fields: `"dock"`

and create `data/presets/fields/dock.json` containing something like:
```
{
    "key": "dock",
    "type": "typeCombo",
    "label": "Type"
}
```

...to let people pick the type of dock..